Source/Core/VideoBackends/OGL/TextureConverter.cpp

```diff
@@ -214,7 +214,7 @@ void Shutdown()
 }
 
 static void EncodeToRamUsingShader(GLuint srcTexture,
-						u8* destAddr, int dstWidth, int dstHeight, int readStride,
+						u8* destAddr, int dstWidth, int dstHeight, int dstStride,
```

Comment:
Applying dstStride as arguments sounds very fine to me, but else I don't see why you need to touch this function at all.
dstHeight=xfb_lines, dstWidth=fb_width/2, readStride=fb_width*2, dstStride=fb_stride*32